lib/pingback.js

```diff
@@ -35,6 +35,7 @@ module.exports = (config) => {
 				result: data.result,
 				action: data.action,
 				url: data.url,
+				'last-modified': data.lastModified,

```

Comment:
its the only one quoted. we should quote all other for consistency

Reply:
or maybe use camel-case?

Reply:
Well, I preferred to use the "spec'ed" spelling...